lib/document.js

```diff
@@ -3912,6 +3927,26 @@ Document.prototype.$toObject = function(options, json) {
   return ret;
 };
 
+/*!
+ * Internal shallow clone alternative to `$toObject()`: much faster, no options processing
+ */
+
+Document.prototype.$__toObjectInternal = function $__toObjectInternal() {
```

Comment:
i dont think `Internal` is a good suffix for this, maybe consider something along the lines of `$__toObjectShallow`?

Reply:
Fixed in 8223f91, good suggestion :+1: